app/code/Magento/Wishlist/Api/WishlistRepositoryInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Magento\Wishlist\Api;
+
+/**
+ * Interface WishlistRepositoryInterface
+ * @api
+ * @package Magento\Wishlist\Api
+ */
+interface WishlistRepositoryInterface
```

Comment:
1. The repository should have methods for CRUD operations: get, getList, save and delete. 
2. Wishlist Repository should manipulate Wishlist entity only 
3. Use strict types for methods